Mathlib/Data/DFinsupp/Lex.lean

```diff
@@ -45,10 +47,17 @@ theorem lex_def {r : ι → ι → Prop} {s : ∀ i, α i → α i → Prop} {a
 instance [LT ι] [∀ i, LT (α i)] : LT (Lex (Π₀ i, α i)) :=
   ⟨fun f g ↦ DFinsupp.Lex (· < ·) (fun _ ↦ (· < ·)) (ofLex f) (ofLex g)⟩
 
+instance [LT ι] [∀ i, LT (α i)] : LT (Colex (Π₀ i, α i)) :=
+  ⟨fun f g ↦ DFinsupp.Lex (· > ·) (fun _ ↦ (· < ·)) (ofLex f) (ofLex g)⟩
+
 theorem lex_lt_iff [LT ι] [∀ i, LT (α i)] {a b : Lex (Π₀ i, α i)} :
     a < b ↔ ∃ i, (∀ j, j < i → a j = b j) ∧ a i < b i :=
   .rfl
 
+theorem colex_lt_iff [LT ι] [∀ i, LT (α i)] {a b : Colex (Π₀ i, α i)} :
```

Comment:
We're converging towards `synonym`'s `op` being referred to as `synonymOp` in lemma names. Could you adapt?
```suggestion
theorem colexLT_iff [LT ι] [∀ i, LT (α i)] {a b : Colex (Π₀ i, α i)} :
```
In fact, here you should probably go even further and move to the `DFinsupp.Colex` namespace:
```suggestion
theorem Colex.lt_iff [LT ι] [∀ i, LT (α i)] {a b : Colex (Π₀ i, α i)} :
```
This is what we do for eg `Sum` and `Prod`

Reply:
I can do this. It will require a bunch of other renames to stay consistent, but I'll put that in a separate PR.